README.md

```diff
@@ -52,13 +77,21 @@ This will build a standalone jar at `backend/build/libs/globe42.jar`, that you c
 
     java -jar backend/build/libs/globe42.jar --globe42.secretKey=<some secret key>
     
-
 To start the application with the demo profile, add this command-line option:  
     
     --spring.profiles.active=demo
     
 And the full app runs on http://localhost:9000
 
+By default, the default GCS credentials are used when launching the app this way. That means
+that the GCS APIs won't be accessible unless you set the `GOOGLE_APPLICATION_CREDENTIALS` as
+described in [the documentation about default credentials](https://developers.google.com/identity/protocols/application-default-credentials#howtheywork]
```

Comment:
last parenthesis is missing : (https://developers.google.com/identity/protocols/application-default-credentials#howtheywork]